sql/core/src/test/scala/org/apache/spark/sql/execution/SQLViewSuite.scala

```diff
@@ -903,4 +903,88 @@ abstract class SQLViewSuite extends QueryTest with SQLTestUtils {
       }
     }
   }
+
+  test("temp view") {
```

Comment:
Ah sorry I pointed to the run test suite. This is the old test suite, we have a new one for view: `SQLViewTestSuite`.

For temp view, the test can be put it `TempViewTestSuite`, which extends `SQLViewTestSuite` and runs with both local and global temp view.

For persistent view, the test can be put it `PersistedViewTestSuite`

Reply:
done.